templates/portfolios/edit_form.html

```diff
@@ -0,0 +1,50 @@
+{% extends "../base.html" %}
+{% load bootstrap3 %}
+
+{% block title %}
+Welcome to Compliance Automation
+{% endblock %}
+
+{% block body-wide %}
+
+<div class="container">
+    <div class="row">
+        <div class="col-md-3 portfolio-org-col ">
+            <h2>Edit Portfolio</h2>
+            <p>Portfolios allow you to organize and manage related projects. Members of a portfolio can access all of
+                its unrestricted projects.</p>
+        </div>
+
+        <div class="col-md-9 portfolio-org-col ">
+            <div class="well">
+                <form method="post">
+                    {% csrf_token %}
+                    <input type=hidden name=action value=neworg>
+                    {% bootstrap_form form %}
+                    <p><button type="submit" id="create-portfolio-button" class="btn btn-success">Update &raquo;</button>
+                        <a type="button" class="btn btn-default" href="/portfolios">Cancel</a>
+                    </p>
+                    
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+{{ block.super }}
+{% endblock %}
+
+{% block scripts %}
+<script>
+    $('#id_name').on('keyup', function () {
```

Comment:
Comments here would help as a quick snapshot of what this is doing.

Reply:
Also, is it necessary to run this code on every key press on this element? It's making everything lowercase on each keypress, but it might be easier to add an on-click event to the "submit" button of the form ("Update") and do this "to lowercase" process once at the end.

Reply:
Looks like this was removed. I'm a fan of that. Resolving this issue.